### PR TITLE
Propagate record timestamps

### PR DESCRIPTION
**Pull request by sbaldwin-rs**
Propagates kafka record timestamps for detection latency calculations

### CODE REVIEW

**Comment by amanpurwar** [comment]
reviewing it now 👀

**Review by amanpurwar — APPROVED** [review]
lgtm!

**Comment by sbaldwin-rs** [comment]
> lgtm!

Great, I will for the backend changes to land before merging.

File: src/main/java/rockset/RocksetRequestWrapper.java [hunk 1]
```diff
@@ -68,6 +69,15 @@ public void addDoc(
                 .key(key)
                 .offset(record.kafkaOffset())
                 .partition(record.kafkaPartition());
+
+        if(record.timestamp() != null){
```
Comment (kwadhwa18) [inline src/main/java/rockset/RocksetRequestWrapper.java:73]: we should add linter to this repo

**Review by kwadhwa18 — APPROVED** [review]

File: src/main/java/rockset/RocksetRequestWrapper.java (hunk 1 above)
Comment (sbaldwin-rs) [reply src/main/java/rockset/RocksetRequestWrapper.java:73]: Created an issue to add this

**Pull request merged**